### ISSUE TITLE
Add Card in Front and remove after down

### ISSUE THREAD

**Issue by lucabecchetti** [tispr/tispr-card-stack#11]
Dear, congrats for your beautiful work! i have a question, is it possible to add new card in front of the current one and not on the bottom?

another question, is it possibile do have a func do delete a card after down effect? something like

``` javascript
func cardDidChangeState(cardIndex: Int) {
        removeFromStack(cardIndex)
}

thanks!
```

**Comment by lucabecchetti** [comment]
Up

**Comment by Pitsko** [comment]
Hi

1) to show new card as first in top stack you should make newCardShouldAppearOnTheBottom = false in layout (property is available in version 0.3.0, published today)

2) to hide cards in bottom stack you can put
        layout.bottomStackMaximumSize = 0
        layout.bottomStackCardHeight = 0
It is workaround, as there are no such functionality as remove cards. probably we will provide it in future

Let me know if you have questions
